{"crankerkor": ["Oleksandr Lutsenko\n", ["\t-nodejs-examples\n", "\t-crankerkor.github.io\n", "\t-lil-trip\n", "\t-PowershapeLab1\n", "\t-discrete-model-labs\n", "\t-da\n"]], "lolegoogle1": ["Oleh Hryshcuk\n", ["\t-lolegoogle1\n", "\t-DjangoProjects\n"]], "LunaLovegoood": ["Loony\n", ["\t-matrix\n", "\t-random-walker\n", "\t-langtons-ant\n", "\t-barnsley-fern\n", "\t-PI-monte-carlo\n", "\t-n-puzzle\n", "\t-PowershapeLab1\n", "\t-simple-game-engine\n", "\t-traffic-flow\n", "\t-matrix-digital-rain\n", "\t-parallel-computing\n", "\t-ahk-scripts\n", "\t-discrete-models-labs\n"]], "Yurii-Khomiak": ["Yurii Khomiak\n", ["\t-dotfiles\n", "\t-yt-audio-playlist-downloader\n", "\t-nvim-config\n"]], "dhmfu": ["Vasil Dudka\n", ["\t-umka-carboncalc\n", "\t-checklist-app\n", "\t-checklist-app-server\n"]], "aasdasdas": ["Noname\n\n", []], "1232": ["Noname\n\n", ["\t-amine\n"]], "0.1231": ["There is no such user, there are only: ", "TypeError Exception"], "^&#61927": ["There is no such user, there are only: ", "TypeError Exception"]}